cli/scripts/monorepo.ts

```diff
@@ -306,7 +306,11 @@ fs.copyFileSync(
     args?: readonly string[],
     options?: execa.SyncOptions<string>
   ) {
-    return execa.sync(turboPath, [command, ...(args || [])], {
+    const resolvedArgs = [...args];
+    if (process.env.TURBO_USE_DAEMON == "1" && command !== "prune") {
```

Comment:
```suggestion
    if (process.env.TURBO_USE_DAEMON == "1" && command === "run") {
```

I think we want just `run`? (In the sense of "allow" vs. "block".)